### ISSUE TITLE
[Feature Request]: Add option to create run configurations

### ISSUE THREAD

**Issue by hansva** [apache/hop#2302]
### What would you like to happen?

You can almost create everything command line, but you cant create/edit run configurations

### Issue Priority

Priority: 3

### Issue Component

Component: Hop Config

**Comment by mattcasters** [comment]
That's right  You can't automtically. You'd need to import metadata or create the JSON file by hand.
